### PR TITLE
Update icon colors

### LINKED ISSUE #32: New default coloring
As TLS 1.0 and 1.1 are getting disabled [(someday)](https://hacks.mozilla.org/2020/02/its-the-boot-for-tls-1-0-and-tls-1-1/), it is time to consider new colors to use to better highlight the differences.

- Ideally, perfect security must be transparent, therefore I suggest using the same gray color for 1.3 as Firefox itself uses for the padlock (black with opacity 0.6).

- Mixed protocols could perhaps use an icon crossed with a red line, similar to the [broken padlock](http://http.badssl.com)). 
A simple line should obstruct the number less than [the previous warning triangle](https://github.com/jannispinter/indicatetls/issues/18).

- Could 1.2 be yellow now? It's not specifically _bad_ until proven that a site uses insecure ciphers and it is still a recommended default. Maybe use a cross for [bad configurations](https://en.wikipedia.org/wiki/Transport_Layer_Security#Cipher) and 1.3 coloring for the rest? 

- 1.0 and 1.1 should be bright red while they still exist, [but is that enough?](https://github.com/jannispinter/indicatetls/issues/19#issuecomment-578430054)

- Wait, why is SSL 3 still in this extension?

- Unknown icon could be inverted, as it doesn't give much info anyway.

### PR DESCRIPTION
**Pull request by Madis0**
Fixes #32

### CODE REVIEW

**Comment by jannispinter** [comment]
We need to update the colors used in the resources tab as well. Those are defined in the `protocolColorMap` in `popup/popup.js`. I'm going to swap them in a minute.

**Comment by jannispinter** [comment]
I left the bright green color for TLSv1.3 in place for the resource tab. What do you think, shall we merge it?

**Comment by Madis0** [comment]
We should remove SSL v3 and readd the warning icon first.

**Comment by jannispinter** [comment]
Thank you @Madis0! 

I will test this via temporary extension side-loading today (`about:debugging#/runtime/this-firefox`). If everything works as expected, I'll release it as version `0.2` tonight.

File: background.js [hunk 1]
```diff
@@ -5,6 +5,13 @@ versionIconMap.set('TLSv1.1', 'icons/tlsv11.png');
 versionIconMap.set('TLSv1', 'icons/tlsv10.png'); 
 versionIconMap.set('unknown', 'icons/tlsunknown.png');
 
+var versionIconWarningMap = new Map();
+versionIconWarningMap.set('TLSv1.3', 'icons/tlsv13_warning.png'); 
+versionIconWarningMap.set('TLSv1.2', 'icons/tlsv12_warning.png'); 
+versionIconWarningMap.set('TLSv1.1', 'icons/tlsv11_warning.png'); 
+versionIconWarningMap.set('TLSv1', 'icons/tlsv10_warning.png'); 
+versionIconWarningMap.set('unknown', 'icons/tlsunknown.png');
```
Comment (Madis0) [inline background.js]: Does unknown need a warning state?

Comment (jannispinter) [reply background.js]: No, I have removed it in 116db37

**Pull request merged**